src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/service/ReviewScheduleService.kt

```diff
@@ -105,8 +106,8 @@ class ReviewScheduleService(
   }
 
   fun getDeadlineDate(educationStartDate: LocalDate): LocalDate {
-    val startDatePlusFive = workingDayService.getNextWorkingDayNDaysFromDate(5, educationStartDate)
-    val pesPlusFive = workingDayService.getNextWorkingDayNDaysFromDate(5, pesContractDate)
+    val startDatePlusFive = workingDayService.getNextWorkingDayNDaysFromDate(REVIEW_DEADLINE_DAYS_TO_ADD, educationStartDate)
```

Comment:
I think its better having a named constant here 👍 ; not just for the configurable nature, and also we dont like magic numbers ... but also its nice and clearly named so its really easy to read and reason about the code 👍 
Nice 💯